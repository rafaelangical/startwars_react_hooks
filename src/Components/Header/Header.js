import React from "react";

const Header = ({ title, className }) => (
  <header className={className} id="header">
    <h1>{title}</h1>
  </header>
);
export default Header;
