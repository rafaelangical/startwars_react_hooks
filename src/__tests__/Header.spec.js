import React from "react";
import renderer from "react-test-renderer";
import Header from "../Components/Header/Header";

it("renders correctly header", () => {
  const tree = renderer
    .create(<Header title="Star Wars" className="header" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
