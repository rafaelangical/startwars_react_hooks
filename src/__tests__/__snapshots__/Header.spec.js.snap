// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly header 1`] = `
<header
  className="header"
  id="header"
>
  <h1>
    Star Wars
  </h1>
</header>
`;
